pkg/packaging/packaging.go

```diff
@@ -301,6 +301,7 @@ func (p *PackageOptions) Build(ctx context.Context, packageWriter io.Writer, tar
 	p.packagekitops = &packagekit.PackageOptions{
 		Name:                     "launcher",
 		Identifier:               p.Identifier,
+		Title:                    "Kolide Endpoint Agent",
```

Comment:
I don't remember the chain that leads here. Is this the default? I want our official builds to be `Kolide Endpoint Agent`, but I want the other builds to be something like `fmt.Sprintf("Launcher agent for %s", p.Identifier)`

Reply:
I didn't do this the way I meant to! I've updated it now so that the default is `Launcher agent for <identifier>`, but that it can be overridden to `Kolide Endpoint Agent` via `PackageOptions.Title` in our own build process.